tests/std/tests/GH_002030_asan_annotate_vector/test.cpp

```diff
@@ -247,12 +323,12 @@ void test_case_insert_n() {
 }
 
 template <class Alloc, int N = 128>
-void test_case_insert_range() {
+void test_insert_range() {
     using T = typename Alloc::value_type;
 
     vector<T, Alloc> v1(1);
     vector<T, Alloc> v2(N);
-    list<T> l(N);
+    input_iterator_tester<T, N> t;

```

Comment:
`r` is the pirate-approved name for test ranges. (No change requested, I'm just getting punchy.)